src/js/actions/index.js

```diff
@@ -13,3 +14,33 @@ export function hideSpinner() {
     payload: false,
   };
 }
+
+export function fetchLocations() {
+  const url = '/openboxes/api/locations';
```

Comment:
One thing I just noticed is that we will probably need to parameterize the "openboxes" in the URL of the APIs since this can change depending on how the application is deployed. I'll create a ticket for the backlog. NOTE: this is not urgent so can wait until we're polishing the finished product.